stan/math/prim/fun/plus.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef STAN_MATH_PRIM_FUN_PLUS_HPP
+#define STAN_MATH_PRIM_FUN_PLUS_HPP
+
+#include <stan/math/prim/meta.hpp>
+
+namespace stan {
+namespace math {
+
+/**
+ * Returns the unary plus of the input.
+ *
+ * @tparam T Type of input.
+ * @param x input.
+ * @return result of unary plus of the input.
+ */
+template <typename T>
+inline plain_type_t<T> plus(const T& x) {
```

Comment:
Use perfect forwarding when returning the argument.

Reply:
Like this?
```cpp
template <typename T>
inline plain_type_t<T> plus(T&& x) {
  return std::forward(x);
}
```

Reply:
Yes, also make the return type `T` - we don't need plus to eval its argument.